benchmarks/applications-service/README.md

```diff
@@ -0,0 +1,39 @@
+# Applications Service Benchmarks
+
+This space has the benchmarks for the `applications-service`.
+
+## How to run benchmarks
+
+- Enter the studio.
+```
+$ hab studio enter
+```
+- Build and start the applications-service.
+```
+[1][default:/src:0]# start_application_service
```

Comment:
This should be `start_applications_service`

Reply:
Yup! I'll accept a suggestion from you, or I'll go just edit it manually. Thanks though for the review!

Reply:
```suggestion
[1][default:/src:0]# start_applications_service
```